damnit/gui/main_window.py

```diff
@@ -944,6 +945,42 @@ def __init__(self, file_path: Path, parent=None):
         self.setCentralWidget(self.text_edit)
         self.resize(1000, 800)
 
+class QtUpdateAgent(UpdateAgent, QtCore.QObject):
```

Comment:
I appreciate this isn't really part of this PR, but is there any reason we need the update sender & update receiver to be one class? It seems like all they share is the topic name, so they could easily be independent.

Reply:
If there's an easy way to avoid multiple inheritance, I generally think it's worth taking. I don't feel strongly enough to hold this up, though.

Reply:
Ah right, that makes sense. I refactored it in 6ca1363.